apps/rush/src/utilities/ChangeManager.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  IChangeInfo,
+  RushConfiguration,
+  RushConfigurationProject
+} from '@microsoft/rush-lib';
+
+import PublishUtilities, {
+  IChangeInfoHash
+} from './PublishUtilities';
+import ChangeFiles from './ChangeFiles';
+import PrereleaseToken from './PrereleaseToken';
+import ChangelogGenerator from './ChangelogGenerator';
+
+export default class ChangeManager {
```

Comment:
+1 I have no idea what any of these functions are supposed to do.  Please add comments

---
In reply to: [123110268](https://github.com/Microsoft/web-build-tools/pull/244#discussion_r123110268) [](ancestors = 123110268)